src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: rgb(255, 255, 255);
+  max-width: 198px;
+  margin-bottom: 500px !important;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__image img {
+  width: 100%;
+}
+
+.card__title {
+  color: rgb(6, 11, 53);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 17px;
+  padding-top: 40px;
+}
+
+.card__code {
+  color: rgb(97, 96, 112);
```

Comment:
Define `rgb(97, 96, 112)` as an SCSS variable to maintain consistency across your styles.